### DISCUSSION TITLE
Create an organization using an existing as the owner

### DISCUSSION THREAD

**Discussion by ashep** [zitadel/zitadel#7561] [Q&A] [answered]
Hi there!

According to the [documentation](https://zitadel.com/docs/apis/resources/admin/admin-service-set-up-org), I must provide the new user information to set it up as an org admin. The question is how I can use an existing user instead of creating a new one. Thank you.

**Comment by hifabienne** [comment, marked as answer]
Hei @ashep 
With the following request it will take the currently authenticated user and add it as an admin:
https://zitadel.com/docs/apis/resources/mgmt/management-service-add-org

**Reply by ashep** [reply]
Thank you!
